modules/gojira-core/src/main/java/com/flipkart/gojira/execute/kafka/DefaultKafkaTestExecutor.java

```diff
@@ -61,6 +62,7 @@ public void execute(
     RecordHeaders recordHeaders = new RecordHeaders();
     requestData.getHeaders().forEach((k, v) -> recordHeaders.add(new RecordHeader(k, v)));
     recordHeaders.add(new RecordHeader(GojiraConstants.TEST_HEADER, testId.getBytes()));
+    recordHeaders.add(new RecordHeader(GojiraConstants.MODE_HEADER, Mode.TEST.name().getBytes()));
```

Comment:
We are sending the header value as bytes. Do we need to convert it to string while reading it in KafkaFilter ?

Reply:
KafkaFilter already receives this data in byte format handles the conversion.